doc/loganalyzer_to_pytest.md

```diff
@@ -0,0 +1,108 @@
+#### Approach 1
+
+#### Requirements
+Move existed loganalyzer to be supported by pytest framework.
+
+#### Overview
+Current loganalyzer implementation consists of the following parts:
+Init - copy loganalyzer.py to the DUT. Add start marker to the syslog.
+
+Analyze - copy loganalyzer.py and all match,expect,ignore files to the DUT. Extract logs from syslog. Add end marker. Analyze logs based on regexp described in match,expect,ignore files.
+Generate summary and result files.
+
+End - Check if "TOTAL MATCHES != 0" then generate system dump. Store system dump, result and summary files to the ansible host.
+
+In proposed approach script copies loganalyzer.py and files with regexp to the DUT and runs syslog analyze on the DUT.
+
+Current loganalyzer implementation description:
+https://github.com/Azure/SONiC/wiki/LogAnalyzer
+
+**Development**
+**Implement:**
+	Module "loganalyzer.py" with class "Loganalyzer".
+	"Loganalyzer" class - executes shell commands on DUT.
+	**"Loganalyzer" class interface:**
+- __init__(match: list, expect: list, ignore: list)

```

Comment:
While creating Loganalyzer instance, it would be better to pass the DUT information to it. DUT information is needed by subsequent methods.